### DISCUSSION TITLE
NM 3.4.4

### DISCUSSION THREAD

**Discussion by fil123** [Universite-Gustave-Eiffel/NoiseModelling#389] [General]
Hi, I need help to install the new version 3.4.4. I set JAVA_HOME (C:\Program Files\Java\jre1.8.0_281), and GEOSERVER_HOME (C:\Users\utente\AppData\Local\Programs\NoiseModelling\geoserver-2.19.2) with WPS extension. I launched C:\Users\utente\AppData\Local\Programs\NoiseModelling\NoiseModelling_3.4.4\bin\startup_windows.bat and the server connection started, but the localhost:9580 connection is refused.
What mistake do I make? 
Thank you
Filippo

**Comment by nicolas-f** [comment]
Hi

You do not need geoserver. NoiseModelling have all files included, please follow the getting started tutorial:

https://noisemodelling.readthedocs.io/en/latest/Get_Started_Tutorial.html

**Reply by fil123** [reply]
Thank you for the reply:
But
"You do not need geoserver" ?
Following the tutorial:
"Windows : you can directly download the executable install file and launch it."
3.4.4 not have an .exe. 
"To launch WPSBuilder, go to http://localhost:9580 using your preferred web browser." don't go.
What are the steps?
Sorry I didn't understand.
Thank you

**Reply by nicolas-f** [reply]
You can 
- use the exe of 3.4.1
- or download NoiseModelling_3.4.4.zip and extract it
